error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -1899,4 +1900,19 @@ Duration after(StepVerifier.LastStep step, Duration duration) {
       return step.verifyTimeout(duration);
     }
   }
+
+  /**
+   * Don't unnecessarily convert {@link Collection} to {@link Stream} before creating {@link Flux}.
+   */
+  static final class FluxFromIterable<T> {
+    @BeforeTemplate
+    Flux<T> before(Collection<T> c) {
```

Comment:
```suggestion
    Flux<T> before(Collection<T> collection) {
```
In most cases, we use the full name `collection` :).